packages/plugins/src/plugins/request.ts

```diff
@@ -0,0 +1,33 @@
+import { PermissionType, PluginFunction } from '@chatbot/bot';
+import fetch from 'node-fetch';
+
+export const calc: PluginFunction = (bot) => {
+    bot.addCommand({
+        name: 'request',
+        args: [],
+        description: 'Sends an HTTP request and gives a response',
+        shortcuts: ['request', 'fetch', 'http'],
+        examples: ['||request [url] [method] [body]', '||request https://httpbin.org/post POST "arg1: value, arg2: value"', '||request https://github.com/ GET'],
+        ignore: false,
+        permissions: [PermissionType.ALL],
+        cb: async (msg, client) => {
+            let params = {
+              method: msg.args[1],
+              headers: {
+                "Content-Type": "application/json"
+              }
+            };
+            if (!(msg.args[0] && msg.args[1])) return client.hardReply('Error: The "url" and "method" arguments are required', msg);
+            if (!msg.args[2]) msg.args[2] = "";
+            if (msg.args[1] !== 'GET') {
```

Comment:
"not GET" does not equal "POST". This will allow other methods and, on top of that, add a JSON body where it's not supposed to.

Reply:
Don't other methods have a body parameter too?

Reply:
Ok, changed `!=== "GET"` to `=== "POST"`